browser/src/control/Control.Sidebar.ts

```diff
@@ -187,10 +193,11 @@ class Sidebar {
 		if (
 			sidebarData.action === 'close' ||
 			window.app.file.disableSidebar ||
-			this.map.isReadOnlyMode()
+			this.map.isReadOnlyMode() ||
+			!(sidebarData.children && sidebarData.children.length)
 		) {
 			this.closeSidebar();
-		} else if (sidebarData.children) {
+		} else {
```

Comment:
best to leave as it was (sidebarData.children check)
not sure we filtered all the actions which will not have that property

Reply:
Agreed, I've pushed the minimal changeset. This file could really use a refactor, but this isn't the commit to do it in (or rather, half-heartedly do a tiny amount in :))